FWCore/Framework/interface/Event.h

```diff
@@ -306,23 +307,23 @@ namespace edm {
     ProductPtrVec putProducts_;
 
     EventAuxiliary const& aux_;
-    mutable std::optional<LuminosityBlock> luminosityBlock_;
+    CMS_SA_ALLOW mutable std::optional<LuminosityBlock> luminosityBlock_;
```

Comment:
Now we should also add the explanation why the `mutable` is allowed  :)

Reply:
What is the comment you would like added. I will add it to the PR.

Reply:
```
    //measurable performance gain by only creating LuminosityBlock when needed
    CMS_SA_ALLOW mutable std::optional<LuminosityBlock> luminosityBlock_;
```

Reply:
Should we add something about `edm::Event` object being intended to be accessed only from one thread?